api/config/config_stable.json

```diff
@@ -0,0 +1,22 @@
+{
+  "firecloud": {
+    "debugEndpoints": false,
+    "billingAccountId": "014D91-FCB792-33D2C0",
+    "billingProjectPrefix": "aou-rw-stable-",
+    "registeredDomainName": "all-of-us-registered-stable",
```

Comment:
We need to make sure this got added as an authorization domain.

Reply:
@blrubenstein Can you handle this? Something like `curl https://api-dot-aou-res-workbench-stable.appspot.com/v1/auth-domain/all-of-us-registered-stable -d ''` I believe.